front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -240,13 +240,13 @@ export default function Project() {
                   </div>
                 </div>
               </div>
-              {(project.funders || (project.budget !== undefined && project.budget !== 0)) && (
+              {(project.funders || (project.budget !== null && project.budget !== undefined && project.budget !== 0)) && (
```

Comment:
As long as we check  for null/undefined/0, you can replace this and the others by : 

```suggestion
              {(project.funders || project.budget) && (
```